src/main/java/org/gatorgradle/task/GatorGradleReportTask.java

```diff
@@ -0,0 +1,25 @@
+package org.gatorgradle.task;
+
+import org.gatorgradle.task.GatorGradleTask;
+import org.gatorgradle.command.BasicCommand;
+import org.gatorgradle.command.Command;
+import org.gatorgradle.config.GatorGradleConfig;
+import org.gatorgradle.display.CommandOutputSummary;
+import org.gatorgradle.internal.DependencyManager;
+import org.gatorgradle.internal.ProgressLoggerWrapper;
+import org.gatorgradle.util.Console;
+
+import org.gradle.api.GradleException;
+import org.gradle.api.tasks.TaskAction;
+import org.gradle.workers.IsolationMode;
+
+public class GatorGradleReportTask extends GatorGradleTask {
+
+  /**
+   * Upload the check report executed by the dependOn GatorGradleGradeTask.
+   */
+  @TaskAction
+  public void report() {
+    summary.uploadOutputSummary();
```

Comment:
Does this work if you run just `gradle report`? Or do you need to do `gradle grade report`? I'm not sure if this will work, did you test it?

Reply:
Also, this should probably also call `super.act()`, if it's designed to work on it's own -- if not (it requires a `gradle grade` run), it shouldn't.

Reply:
Ah, ignore these previous comments -- I see the `dependsOn` relationship and I think this should work as intended. Just needs testing!